src/core/main.c

```diff
@@ -1679,7 +1679,7 @@ int main(int argc, char *argv[]) {
         log_open();
 
         if (arg_show_status == _SHOW_STATUS_UNSET)
-                arg_show_status = SHOW_STATUS_YES;
+                arg_show_status = arg_confirm_spawn ? SHOW_STATUS_NO : SHOW_STATUS_YES;
```

Comment:
That sounds good.

I gave it a try and one minor issue is that if the message line doesn't fit the screen width then the message is reprinted to next line every time.

But otherwise this improves the overall usability when the confirm messages share the same console with the logs.